src/box.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {BoxProps} from './types/box-types'
+import {propTypes} from './enhancers'
+import enhanceProps from './enhance-props'
+
+export default class Box extends React.Component<BoxProps, {}> {
```

Comment:
Would it make more sense to export this as a `React.SFC` or `React.FC` instead of a class?

Reply:
Maybe so. I was attempting to limit the scope of change as much as possible since this is a massive migration.